Convert Infix Expressions to Postfix Expression

Screen-sLaYeR opened the pull request:
#### Description of Change

Added a infix to postfix converter's code

Panquesito7 reviewed (changes requested):
Hey there! Thank you for submitting a PR! 😄 Sadly, this does not meet our requirements. Check out our [repository standards](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md). Please read them **carefully** and **follow them**. **Let us know if you need any help here or on our [Discord server](https://the-algorithms.com/discord).** 🙂

Screen-sLaYeR commented:
#### Description of Change
Added a new Code to Convert Infix Expressions to Postfix Expressions using Stack.
<!--
Thank you for your Pull Request. Please provide a description above and review
the requirements below.

Contributors guide: https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md
-->


<!-- Add any reference to a previous pull request or issue -->

#### Checklist
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [x] Added description of the change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#File-Name-guidelines)
- [x] Added tests and examples, the test must pass
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes: <!-- Please add a one-line description for developers or pull request viewers -->
